nixnet/database/database.py

```diff
@@ -16,7 +16,18 @@
 
 
 class Database(object):
-
+    """Opens a database file.
+
+    When an already open database is opened,
+    this class grants access to the same database and increases an internal reference counter.
+    A multiple referenced (open) database must be closed as many times as it has been opened.
```

Comment:
"A multiple referenced (open) database must be closed as many times as it has been opened."
Should we capture this as a Note in the docstring? And explain that in python upon `database ` object exit, `close ` will get called automatically for them.